### PR TITLE
fixed user service giving long stack trace error

### PR DESCRIPTION
**Pull request by ethirocks**
https://trello.com/c/3UvvAfQ4/37-fix-userserviceverifyemail

### CODE REVIEW

**Review by LinusCastelino — APPROVED** [review]
Looks good..
Please test once merged into master

File: src/main/react-front-end/src/APICalls/APICalls.js [hunk 1]
```diff
@@ -59,12 +59,12 @@ export async function checkLogin(email, accept, fail){
...
-		statusHandle(response, callback);
-	}).catch((error) => {
-      statusHandle(error, fail);
-    });
+		if((response.data === true)){
+			statusHandle(response, accept);
+		}else if(response.data === false){
+			statusHandle(response, fail);
+		}
+	});
```
Comment (aashish-jain) [inline src/main/react-front-end/src/APICalls/APICalls.js:67]: What if backend throws an error due to some server issues. So, other errors have to be handled. Also. I think the error handling block is still needed.

Comment (LinusCastelino) [reply src/main/react-front-end/src/APICalls/APICalls.js:67]: If there is an error, the previous method would also behave the same way.
And if the server returns false, the UI is already configured to say 'Email not found or server error.'
In this scenario, only database read can be an issue, and if that happens, the framework will bombard server logs with errors.

**Comment by LinusCastelino** [comment]
[Fix UserService.verifyEmail()](https://trello.com/c/3UvvAfQ4/37-fix-userserviceverifyemail)

File: src/main/react-front-end/src/APICalls/APICalls.js (hunk 1 above)
Comment (LinusCastelino) [reply src/main/react-front-end/src/APICalls/APICalls.js:67]: Agreed that there must be a catch block regardless.
Reverting the PR
@ethirocks please handle this

**Pull request merged**